hashicorp/vault
# Backport of fix: Broken docs links for Auto-Auth page into release/1.14.x

## Backport

This PR is auto-generated from #21938 to be assessed for backporting due to the inclusion of the label backport/1.14.x.



The below text is copied from the body of the original PR.

---

[:mag: Preview Link for Agent page](https://vault-git-nq-fix-autoauth-docs-links-hashicorp.vercel.app/vault/docs/agent-and-proxy/agent)
[:mag: Preview Link for API proxy page](https://vault-git-nq-fix-autoauth-docs-links-hashicorp.vercel.app/vault/docs/agent-and-proxy/agent/apiproxy)

---

This PR removes `agent` from links to the Auto-Auth docs page, fixing the links.

### Verification

1. Click each of the preview links above.
2. Press `Ctrl+F` and search for "Auto-Auth" to find the relevant links. 
3. Click these links. They should successfully navigate you to [the Auto-Auth page](https://vault-git-nq-fix-autoauth-docs-links-hashicorp.vercel.app/vault/docs/agent-and-proxy/autoauth).

---

<details>
<summary> Overview of commits </summary>

  - 053349f7712d9b4dd706c61db2d62c2f54e46801 

</details>
